src/libcrun/linux.c

```diff
@@ -1111,7 +1111,7 @@ do_masked_or_readonly_path (libcrun_container_t *container, const char *rel_path
     {
       ret = get_file_type_fd (pathfd, &mode);
       if (UNLIKELY (ret < 0))
-        return ret;
+        return crun_make_error (err, errno, "cannot stat fd");
```

Comment:
could we add `rel_path` to the error message string?

Reply:
I replaced fd with `rel_path`
Does it look ok now?

Reply:
yes!